package.json

```diff
@@ -185,6 +185,7 @@
         "npm-run-all2": "^6.0.0",
         "patch-package": "^8.0.0",
         "pinst": "^3.0.0",
+        "plop": "^4.0.1",
```

Comment:
This doesn't need to be a root-level dependency. It's a dependency of the templates package and is run in that workspace.